Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -397,3 +397,29 @@ theorem tprod_const [T2Space G] (a : G) : ∏' _ : β, a = a ^ (Nat.card β) :=
       simpa [multipliable_const_iff] using ha
 
 end IsTopologicalGroup
+
+section CommGroupWithZero
+variable {K : Type*} [CommGroupWithZero K] [TopologicalSpace K] [ContinuousMul K]
+
+open Finset in
+lemma Multipliable.congr_cofinite'
+    {f g : α → K} (hf : Multipliable f) (hf' : ∀ a, f a ≠ 0)
+    (hfg : ∀ᶠ a in cofinite, f a = g a) : Multipliable g := by
```

Comment:
Can we have a `HasProd` and `tsum` version of this too? That is, if `HasProd f c` (and f is nonzero and cofinitely often equal to g) then `HasProd g c` too, and under the same conditions `tsum f = tsum g`.

Reply:
> That is, if HasProd f c (and f is nonzero and cofinitely often equal to g) then HasProd g c too

That isn't true; if you change finitely many terms of a product you certainly also change the limit. 

The closest HasProd analogue I can think of for this would be something like
```
lemma HasProd.congr_cofinite'
    {f g : α → K} {c : K} (hf : HasProd f c) (hf' : ∀ a, f a ≠ 0)
    (s : Finset α) (hs : ∀ a ∉ s, f a = g a) :
     HasProd g (c * ((∏ i ∈ s, g i) / ∏ i ∈ s, f i) := by
  sorry
```
Shall I add this?

Reply:
Oops, you're of course correct. That lemma looks helpful, thank you!